frontend/src/metabase/search/components/SearchSidebar/tests/common.unit.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import { screen } from "__support__/ui";
+import { setup } from "./setup";
+
+describe("SearchFilterSidebar", () => {
```

Comment:
Not sure what else to put in here, since `SearchSidebar` has become a controlled component, and we test the get/set behavior of filters in `SearchApp` and in each individual filter, so not sure if we have a redundant test here. The SearchSidebar is more concerned about controlling where the layout and presence of the filters